featureswitch/featureswitch.go

```diff
@@ -57,6 +57,22 @@ func matchesFeatureForString(featureSwitch string, data string) bool {
 	return matchesFeatureForHash(featureSwitch, hash)
 }
 
+func setFeatureSwitchLevel(featureSwitch string, level float64) func() {
+	oldValue, wasSet := os.LookupEnv(featureSwitch)
+	os.Setenv(featureSwitch, strconv.FormatFloat(level, 'f', 4, 64))
```

Comment:
Is it better to use `strconv.FormatFloat` than `fmt.Sprintf("%f")`?

Reply:
No, I think Sprintf is honestly better.